scripts/sql/166_helm_async_install.up.sql

```diff
@@ -0,0 +1,3 @@
+alter table installed_app_version_history
+    add column helm_release_status_config text;
```

Comment:
we can add description, that will help to understand this json for status config